pkg/registry/batch/job/strategy_test.go

```diff
@@ -294,142 +411,21 @@ func TestJobStrategy_PrepareForCreate(t *testing.T) {
 
 			Strategy.PrepareForCreate(ctx, &tc.job)
 
+			if tc.job.Status.Active != 0 {
+				t.Errorf("Job does not allow setting status on create")
+			}
+			if tc.job.Generation != 1 {
+				t.Errorf("expected Generation=1, got %d", tc.job.Generation)
+			}
+
```

Comment:
Are these checks necessary? It seems changes here would be picked by the `cmp.Diff` below

Reply:
Yea I think you may be right. I’ll test and see.

Reply:
Removed.